modules/ROOT/pages/update-workspace.adoc

```diff
@@ -71,6 +72,26 @@ Do not to preserve the following declarations:
 * `classLoaderModelLoaderDescriptor`
 * `bundleDescriptorLoader`
 
+[[exchange-dependency-update]]
+== Exchange Update for Workspaces Created With Studio 7.11.x and Earlier Versions
+
+Because Anypoint Studio 7.12 adds support for comprehensive publication of assets to Exchange, your projects must add a dependency to the new version of Anypoint Exchange.
```

Comment:
I'm not sure about using `dependency` here, I would change this to `repository`. Feels more accurate

Reply:
The possessive seems to be inappropriate here: "...to identify the application's classloader configuration and settings...". It should read "...to identify the application classloader configuration and settings..."